Sources/Controllers/MessagesViewController.swift

```diff
@@ -47,7 +47,7 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
     /// bottom whenever the `InputTextView` begins editing.
     ///
     /// The default value of this property is `false`.
-    /// NOTE: This calls scrollToBottome where as the above flag calls  scrollToLastItem - check methods for differences
+    /// NOTE: This calls scrollToBottom where as the above flag calls  scrollToLastItem - check methods for differences
```

Comment:
Can you update this comment to what I commented on your previous PR? (Fix extra space, other wording)

Reply:
Updated